docs/reference/spark-client.md

```diff
@@ -102,7 +102,7 @@ Include this assembled jar (an "Überjar") from S3, from
     */
    ```
 
-1. Run SQL queries on your metadata:
+2. Run SQL queries on your metadata:
```

Comment:
This is actually intentional: writing consecutive numbers makes trivial code reviews substantially harder.  If you start each line with "1." then it's more like an HTML <li> inside <ol>, and we never need to fix numbers when you insert something.  That makes diffs shorter & clearer.